extensions/amp-sidebar/1.0/amp-sidebar.css

```diff
@@ -10,6 +10,8 @@ amp-sidebar {
   background-color: #efefef;
   height: 100vh;
   top: 0;
+  outline: none;
+  z-index: 2147483647;
```

Comment:
I did not find any other instance in our code base where we set z-index this high. Not sure if we want to start some kind of z-index arms race 🙂

Reply:
This value was set when the component styling was done initially [here](https://github.com/ampproject/amphtml/commit/7311ed960f9aa71e5160095e3cb64db75fa037f6#diff-ff22105e57037561cfba33a82c88dcf4243939ddcc5318ed895143510eafe913R30-R31)

I believe this is to make sure that any sidebar displays on top of anything on the page.